Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:14-alpine@sha256:5c33bc6f021453ae2e393e6e20650a4df0a4737b1882d389f17069dc1933fdc5
+FROM node:lts-alpine@sha256:f4c96a28c0b2d8981664e03f461c2677152cd9a756012ffa8e2c6727427c2bda
```

Comment:
could you write the exact version of this in a comment so we don't have to dig around trying to work it out.  We have had issues in the past where the node engine is different to the Docker version

Reply:
Have specified using node version 20